src/index.html

```diff
@@ -7,7 +7,58 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="body">
+
+    <div class="main-container">
+      
+      <div class="container stars--0">
+        <div class="star"></div>
+        <div class="star"></div>
+        <div class="star"></div>
```

Comment:
It's a wrong class names due to BEM methodology
There is a block (`stars`) and it has elements (`stars__star`)
It's okay now, but when you will reuse this block, you'll have to rename classes